components/popover/index.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import Tooltip from '../tooltip';
 import { AbstractTooltipProps } from '../tooltip';
 import warning from '../_util/warning';
+import { TooltipPlacement } from '../tooltip';
```

Comment:
I think this import should be placed in the line 3, e.g. `import { AbstractTooltipProps, TooltipPlacement } from '../tooltip'`